openmetadata-docs/content/openmetadata/connectors/metadata/atlas/index.md

```diff
@@ -0,0 +1,160 @@
+you can find the structure to create a connection to Atlas.
+
+In order to create and run a Metadata Ingestion workflow, we will follow the steps to create a
+YAML configuration able to connect to the source, process the Entities if needed, and reach the OpenMetadata server.
+
+The workflow is modeled around the following [JSON Schema](https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-spec/src/main/resources/json/schema/entity/services/connections/metadata/atlasConnection.json).
+
+### 1. Define the YAML Config
+
+This is a sample config for Atlas:
+
+```yaml
+source:
+  type: Atlas
+  serviceName: local_atlas
+  serviceConnection:
+    config:
+      type: Atlas
+      atlasHost: http://192.168.1.8:21000
+      username: admin
+      password: admin
+      dbService: Hive
+      messagingService: Mysql
+      serviceType: Hive
+      hostPort: localhost:10000
+      entityTypes: examples/workflows/atlas_mapping.yaml
+  sourceConfig:
+    config:
+      type: DatabaseMetadata
+sink:
+  type: metadata-rest
+  config: {}
+workflowConfig:
+  openMetadataServerConfig:
+    hostPort: http://localhost:8585/api
+    authProvider: no-auth
+```
+
+This is a sample config for Atlas mapping:
+
+```yaml
+Table:
+  rdbms_table:
+    db: rdbms_db
+    column: rdbms_column
+Topic:
+  - kafka_topic
+  - kafka_topic_2
+```
+
+### Source Configuration - Service Connection
+
+You can find all the definitions and types for the `serviceConnection` [here](https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-spec/src/main/resources/json/schema/entity/services/connections/metadata/atlasConnection.json).
+
+- `username`: Username to connect to the Atlas. This user should have privileges to read all the metadata in Atlas.
+- `password`: Password to connect to the Atlas..
+- `hostPort`: Host and port of the data source..
+- `entityTypes`: entity types of the data source.
+- `serviceType`: service type of the data source..
+- `atlasHost`: Atlas Host of the data source.
+- `dbService` (Optional): source database of the data source.
+- `messagingService` (Optional): messaging service source of the data source.
+- `database` (Optional) :Database of the data source. This is optional parameter, if you would like to restrict the metadata reading to a single database. When left blank , OpenMetadata Ingestion attempts to scan all the databases in Atlas.
+
+### Sink Configuration
+
+To send the metadata to OpenMetadata, it needs to be specified as `"type": "metadata-rest"`.
+
+### Workflow Configuration
+
+The main property here is the `openMetadataServerConfig`, where you can define the host and security provider of your
+OpenMetadata installation. For a simple, local installation using our docker containers, this looks like:
+
+```yaml
+workflowConfig:
+  openMetadataServerConfig:
+    hostPort: http://localhost:8585/api
+    authProvider: no-auth
+```
+
+### OpenMetadata Security Providers
```

Comment:
I've updated the security config and examples with this [pr](https://github.com/open-metadata/OpenMetadata/pull/7877)  let's follow the same here as well